### PR TITLE
[INF] Fix Mizu theme display problems with CJK font

### PR DESCRIPTION
**Pull request by njlyf2011**
## Purpose

JIRA issue: [CORE-17997](https://jira.reactos.org/browse/CORE-17997)

### CODE REVIEW

**Comment by njlyf2011** [comment]
Before and After:
Before
After

**Comment by JoachimHenze** [comment]
Can you please post a before-and-after-picture of **english** ReactOS with the Mizu theme, as well?
We don't want the "Droid Sans Fallback" font to be used then.

**Comment by njlyf2011** [comment]
> Can you please post a before-and-after-picture of **english** ReactOS with the Mizu theme, as well? We don't want the "Droid Sans Fallback" font to be used then.

If you did not select Chinese, Japanese and Korean during installation, then this PR will have no effect on your OS font settings.

image

File: media/inf/font.inf [hunk 1]
```diff
@@ -86,6 +86,7 @@ HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","MS Shell Dl
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","MS Shell Dlg 2",0x00000000,"Droid Sans Fallback"
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","MS UI Gothic",0x00000000,"Droid Sans Fallback"
 HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","MS UI Gothic 2",0x00000000,"Droid Sans Fallback"
+HKLM,"SOFTWARE\Microsoft\Windows NT\CurrentVersion\FontSubstitutes","Source Sans Pro",0x00000000,"Droid Sans Fallback"
```
Comment (katahiromz) [inline media/inf/font.inf:89]: Latin user might want to use original `"Source Sans Pro"`...
Can you make it more user-language-specific?

Comment (katahiromz) [reply media/inf/font.inf:89]: Use `base/setup/lib/muifonts.h`.

**Review by katahiromz — APPROVED** [review]
Okay. The problem and solution are limited in the CJK environment. No change against non-CJK.

**Comment by binarymaster** [comment]
@foxlet it's about your theme, can you please check this PR? Does it look as expected?

**Comment by njlyf2011** [comment]
@katahiromz 
/base/setup/lib/muifonts.h is updated.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**